src/plone/restapi/tests/test_serializer_converters.py

```diff
@@ -136,11 +142,11 @@ def test_persistent_mapping(self):
         value = PersistentMapping({'foo': 'bar'})
         self.assertEquals({u'foo': u'bar'}, json_compatible(value))
         self.assertEquals('{"foo": "bar"}', json.dumps(json_compatible(value)))
-        self.assertIsInstance(json_compatible(value.keys()[0]),
-                              unicode,
+        self.assertIsInstance(json_compatible(list(value.keys())[0]),
```

Comment:
this should be written: `self.assertIsInstance(json_compatible(list(value)[0]),`